pkgs/by-name/dr/dreamchess/package.nix

```
@@ -0,0 +1,39 @@
+{ lib, stdenv, fetchFromGitHub,
+  cmake, bison, flex, gettext, SDL2, SDL2_image, SDL2_mixer, expat, glew,
+  freetype, libSM, libXext, libGL, libGLU, xorg
+ }:
+
+stdenv.mkDerivation rec {
+  pname = "dreamchess";
+  version = "0.3.0";
+  src = fetchFromGitHub {
+    owner = "dreamchess";
+    repo = "dreamchess";
+    rev = "${version}";
+    hash = "sha256-qus/RjwdAl9SuDXfLVKTPImqrvPF3xSDVlbXYLM3JNE=";
+  };
+
+  buildInputs = [
+    SDL2 SDL2_image SDL2_mixer expat glew freetype
+    libSM libXext libGL libGLU xorg.libxcb xorg.libX11
+  ];
+  nativeBuildInputs = [
+    cmake
+    bison
+    flex
+    gettext
+  ];
+  cmakeFlags = [
+    (lib.cmakeBool "CMAKE_VERBOSE_MAKEFILE" true)
+    (lib.cmakeFeature "OpenGL_GL_PREFERENCE" "GLVND")
+    (lib.cmakeFeature "CMAKE_INSTALL_DATAROOTDIR" "${placeholder "out"}/share")
+  ];
+
+  meta = {
+    homepage = "https://github.com/dreamchess/dreamchess";
+    description = "DreamChess - OpenGL Chess Game";
+    license = lib.licenses.gpl3;
```

```suggestion
    license = lib.licenses.gpl3Only;
```

https://github.com/dreamchess/dreamchess/blob/12c45028a8a57d4ef751fbad25f6434646096701/README.md?plain=1#L17